cypress/integration/cucumber/step_definitions/refapp-3.x/04-clinical-visit/clinical-visit.js

```diff
@@ -109,6 +110,34 @@ Then('the added condition should be listed', () => {
     cy.contains('Fever');
 });
 
+Then('the test results should be shown', () => {
+    cy.contains('Viral load');
+})
+
+When('the user clicks on timeline of a test', () => {
+    cy.contains('Timeline').click({force: true});
+})
+
+Then('the timeline table should be shown', () => {
+    cy.contains('copies/ml');
+})
+
+When('the user clicks on trend of a test', () => {
+    cy.contains('Trend').click({force: true});
+})
+
+Then('the trend line should be shown', () => {
+    cy.contains('Back to timeline');
```

Comment:
Is there any other good method to verify whether the trend line is being rendered?